pkg/operation/botanist/component/metricsserver/metrics_server.go

```diff
@@ -76,13 +75,15 @@ func New(
 	image string,
 	vpaEnabled bool,
 	kubeAPIServerHost *string,
+	sideCar string,
```

Comment:
You can give better name.

```suggestion
	addonResizerImage string,
```